### PR TITLE
Info about downloading RAG

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Info about downloading RAG

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [ ] CI configuration change
- [ ] Konflux configuration change

### CODE REVIEW

**Comment by tisnik** [comment]
/override "ci/prow/images"
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

File: README.md [hunk 1]
```diff
@@ -394,10 +394,15 @@ Depends on configuration, but usually it is not needed to generate or use API ke
...
 
 ## 5. (Optional) Configure the local document store
+
+   The following command downloads a copy of the whole image containing RAG embedding model and vector database:
+
    ```sh
    make get-rag
    ```
 
+   Please note that link to the specific image to be downloaded is stored in file `build.args` (and that file is autoupdated by bots when new RAG is re-generated):
```
Comment (JoaoFula) [inline README.md:404]: Please note that **the** link ...

Comment (JoaoFula) [reply README.md:404]: is stored in **the** file `build.args`

Comment (JoaoFula) [reply README.md:404]: by bots when **a** new RAG

Comment (tisnik) [reply README.md]: fixed, thank you!

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**